actions/transactions/search.go

```diff
@@ -51,3 +51,55 @@ func (a *Action) search(c *gin.Context) {
 
 	c.JSON(http.StatusOK, contracts)
 }
+
+// TODO: this method is not finished and will be changed based on search poc
+// transactions will fetch a list of transactions filtered on conditions and metadata
+// Get transactions godoc
+// @Summary		Get transactions
+// @Description	Get transactions
+// @Tags		New Transactions
+// @Produce		json
+// @Param		SearchTransactions body filter.SearchTransactions false "Supports targeted resource searches with filters and metadata, plus options for pagination and sorting to streamline data exploration and analysis"
+// @Success		200 {object} []models.Transaction "List of transactions"
```

Comment:
I believe that the model should be different here

Reply:
Correct. Changed to `models.PageModel[models.Transaction]`